sakura_core/doc/CEditDoc.cpp

```diff
@@ -199,8 +189,7 @@ CEditDoc::CEditDoc(CEditApp* pcApp)
 #ifdef _DEBUG
 	{
 		// 編集禁止コマンドの並びをチェック
-		int i;
-		for ( i = 0; i < _countof(EIsModificationForbidden) - 1; i++){
+		for (auto i = 0; i < _countof(EIsModificationForbidden) - 1; i++){
```

Comment:
指摘ではありません。

せっかく直すならこんな感じが好みです。

```suggestion
		for (size_t i = 0; i < std::size(EIsModificationForbidden); ++i) {
```

拡張 for に移行すると「より良い」はず。